python/cudf/cudf/core/column/column.py

```diff
@@ -97,6 +97,10 @@ class ColumnBase(Column, Serializable, BinaryOperand, Reducible):
 
     _PANDAS_NA_REPR = str(pd.NA)
 
+    @cached_property
+    def nan_count(self) -> int:
```

Comment:
https://github.com/rapidsai/cudf/pull/17765 has been merged so this can be removed now

Reply:
Done